habitat-lab/habitat/config/benchmark/rearrange/rearrange_easy_human_and_spot.yaml

```diff
@@ -0,0 +1,62 @@
+# @package _global_
+
+defaults:
+  - /habitat: habitat_config_base
+  - /habitat/simulator/agents@habitat.simulator.agents.agent_0: depth_head_agent
+  - /habitat/simulator/agents@habitat.simulator.agents.agent_1: rgbd_head_rgbdp_arm_agent
+  # Note, this is a copy of task human_and_fetch config
```

Comment:
Should we have a separate task config for each human & robot combination or just one config to work for all, say, rearrange_easy_human_and_robot config?

Reply:
I'm not an expert at proper config usage in Hab-lab, but my impression is that each robot will need separate configs, e.g. different sensors. Is there a better way to specify configs for these two agents? Can we somehow provide multiple configs at runtime instead of having to author these awkward two-agent configs?

Reply:
Yes, I can work on refactoring the configs to be able to reconfigure agents at runtime. But from what I see, creating a separate and/or two-agent configs is used on main and SIRo. If Jimmy approves these changes I'd suggest to merge this PR and create new minor subtask to refactor the configs (as it may take a quite a bit of time to do)

Reply:
Agree, add the task to the design doc but let's call it low-priority. I'm sure we have higher-pri stuff aimed at unblocking our data-collection.